routes/licenses.js

```diff
@@ -51,7 +44,19 @@ routes.push({
       }),
     },
   },
-  handler: async (request, h) => h.response(mockResponse[0]),
+  handler: async (request, h) => {
+    const { fileData, licenses } = request.server.app.services;
+    const { file } = request.params;
```

Comment:
as far as I understood, the `file` param can contain a url or fileIdentifier -- given that, I find the variable name `file` slightly confusing. Maybe `fileIdentifier` or `fileUrlOrIdentifier` is better (but longer)?